c_emulator/riscv_sim.c

```diff
@@ -416,22 +431,28 @@ void init_spike(const char *f, uint64_t entry, uint64_t ram_size)
   s = tv_init(isa, ram_size, 1);
   if (tv_is_dirty_enabled(s) != rv_enable_dirty_update) {
     mismatch = true;
-    fprintf(stderr, "inconsistent enable-dirty-update setting: spike %s, sail %s\n",
+    fprintf(stderr,
+            "inconsistent enable-dirty-update setting: spike %s, sail %s\n",
             tv_is_dirty_enabled(s) ? "on" : "off",
             rv_enable_dirty_update ? "on" : "off");
   }
   if (tv_is_misaligned_enabled(s) != rv_enable_misaligned) {
     mismatch = true;
-    fprintf(stderr, "inconsistent enable-misaligned-access setting: spike %s, sail %s\n",
-            tv_is_misaligned_enabled(s) ? "on" : "off",
-            rv_enable_misaligned        ? "on" : "off");
+    fprintf(
```

Comment:
This change is a bit odd, might need to manually split the string literal to make clang-format happy.

Reply:
Yes it looks like with 80 cols limit these calls won't fit so clang-format does some slightly odd rewrapping.